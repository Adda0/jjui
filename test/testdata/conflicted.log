@  t;tlosykyx;updates-1;true;false;false;false;ibrahim@dursun.cc;1 week ago;add readme
│ ×  k;klywuoyq;.;false;false;true;false;ibrahim@dursun.cc;1 week ago;add new file
├─╯
×  v;vqqrnsqx;main;false;false;true;false;ibrahim@dursun.cc;1 week ago;add another line
◆  z;zzzzzzzz;.;false;true;false;true;55 years ago
