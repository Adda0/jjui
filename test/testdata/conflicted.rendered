○  tl
×  k
│ @  p
│ ×  tm
├─╯
×  v
◆  z
