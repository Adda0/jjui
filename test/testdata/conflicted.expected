@  tlosykyx
│  add readme
│ ×  klywuoyq
├─╯  add new file
×  vqqrnsqx
│  add another line
◆  zzzzzzzz root()